### PR TITLE
I-D text update resolving #145, #153, and #169

### PR DESCRIPTION
**Pull request by dieterbeller**
This I-D update resolves:

- #145 
- #153 
- #169 

Additional changes:

- Organization updated for @ggalimba56
- @manzoro added to the list of additional authors in the body of the document.
- References updated based on new security considerations.

### CODE REVIEW

**Review by sergiobelotti — APPROVED** [review]

**Review by manzoro — APPROVED** [review]
Fine for me

File: I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml [hunk 1]
```diff
@@ -4859,6 +4957,108 @@ module ietf-optical-impairment-topology {
...
+   defined in <xref target="RFC8795"/> and consequently inherits
+   te-topology subtrees and data nodes and their potential
+   sensitivities and vulnerabilities. Therefore, the security
+   considerations from <xref target="RFC8345"/> also apply to the YANG
+   module defined in this document.
+  </t>
+  
+  <t>
+   The data nodes in this YANG module are not writable nor creatable
+   nor deletable (i.e., config true; the data nodes are readable only
```
Comment (EstherLerouzic) [inline I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml:4993]: miss a parenthesis : (i.e., config true ")"

**Review by EstherLerouzic — COMMENTED** [review]
a minor typo, looks good to me

File: I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml (hunk 1 above)
Comment (dieterbeller) [reply I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml:4993]: Good catch! Already fixed in my local repo.

File: I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml [hunk 2]
```diff
@@ -1332,6 +1426,23 @@
...
+   egress amplifier = input of the fiber span) and an ingress amplifier
+   power measurement facility (power measured at the input of the
+   ingress amplifier in the next node = output of the fiber span).
+   This measured loss typically differs from the calculated loss based
+   on loss-coef and length as it includes all loss contributions
+   including possible accumulated loss due to imperfect fiber splices
+   and connector losses. In case the total-loss cannot be measured (no
+   power measurement facilities in place), this optional attribute
+   shall be omitted.
+  </t>
```
Comment (ju7ien) [inline I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml:1446]: Cf. my 2 comments in issue #169

File: I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml [hunk 3]
```diff
@@ -4859,6 +4971,108 @@ module ietf-optical-impairment-topology {
...
+   defined in <xref target="RFC8795"/> and consequently inherits
+   te-topology subtrees and data nodes and their potential
+   sensitivities and vulnerabilities. Therefore, the security
+   considerations from <xref target="RFC8345"/> also apply to the YANG
+   module defined in this document.
+  </t>
+  
+  <t>
+   The data nodes in this YANG module are not writable nor creatable
+   nor deletable (i.e., config true) the data nodes are readable only
```
Comment (ju7ien) [inline I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml:5007]: "(i.e. not config true)" would be clearer. A semicolon after the bracket would be welcome.

**Review by ju7ien — APPROVED** [review]
Addressing my comments shouldn't make us miss the Monday deadline.

**Review by ggalimba56 — APPROVED** [review]

**Review by aguoietf — APPROVED** [review]

**Review by sergiobelotti — APPROVED** [review]

**Review by manzoro — APPROVED** [review]

File: I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml [hunk 4]
```diff
@@ -4859,6 +5065,108 @@ module ietf-optical-impairment-topology {
...
+   (i.e., config false, which is the default). Although the data nodes
+   are not susceptible to write operations or attacks
+   (e.g., edit-config), network information may be exposed using read
+   access (e.g., get, get-config, or notification) operations. Thus,
+   controlling read access to specific data nodes containing sensitive
+   or vulnerable information is important. Below are the specific
+   subtrees, data nodes and subsequent sensitivity/vulnerability:
+  </t>
```
Comment (italobusi) [inline I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml:5109]: I would suggest to re-use as much as possible the text in section 3.7.1 of RFC8407:

```suggestion
  <t>
    There are no data nodes defined in this YANG module that are
    writable/creatable/deletable (i.e., "config true", which is the
    default).
  </t>

  <t>
    Some of the readable data nodes in this YANG module may be considered
    sensitive or vulnerable in some network environments.  It is thus
    important to control read access (e.g., via get, get-config, or
    notification) to these data nodes.  These are the subtrees and data
    nodes and their sensitivity/vulnerability:
  </t>
```

**Pull request merged**